dali/python/nvidia/dali/pipeline.py

```diff
@@ -1203,6 +1244,22 @@ def iter_setup(self):
         data from NumPy arrays."""
         pass
 
+    def _generate_build_args(self, output_dtype, output_ndim):
+        ret = []
+        num_outputs = len(self._names_and_devices)
```

Comment:
Just curious why do you pass output_dtype and output_ndim but not _names_and_devices as arguments? (small nitpick).

Reply:
Good point